repos/common/actors/augeasscanner/actor.py

```diff
@@ -0,0 +1,68 @@
+from leapp.actors import Actor
+from leapp.models import AugeasInput
+from leapp.models import AugeasOutput, LensesList, LensItem
+from leapp.tags import ScanTag
+
+import augeas
+import json
+
+import augeasprivate
+
+
+class AugeasScanner(Actor):
+    name = 'augeas_scanner'
+    description = 'Actor scans config files using Augeas'
+    consumes = (AugeasInput,)
+    produces = (AugeasOutput,)
+    tags = (ScanTag,)
+
+    def __init__(self, **kwargs):
+        super(AugeasScanner, self).__init__(**kwargs)
+
+        # path to folder with custom lenses
+        self.lenses_folder = 'files/lenses'
+
+        self.augeas_tree = augeas.Augeas()
+
+    def process(self):
+        self.log.info("Starting augeas scanner")
+
+        # for msg in self.consume(AugeasInput):
+        #     self.consumed_msg = msg
+        #     self.log.info("Starting to process...")
+        try:
+            self.consumed_msg = self.consume(AugeasInput).next()
```

Comment:
1. No sense to use member variables for this on self, there's no use for these
2. Use next(self.consume(AugeasInput), None) which will give you None as a result if there's no message, that will help you to avoid having to process the input.
```python
consumed_msg = next(self.consume(AugeasInput), None)
if consumed_msg:
    ...
```